client/my-sites/importer/importer-header.jsx

```diff
@@ -71,15 +71,15 @@ export const ImporterHeader = React.createClass( {
 		const { importerState } = this.props.importerStatus;
 
 		if ( includes( startStates, importerState ) ) {
-			return this.translate( 'Start Import', { context: 'verb' } );
+			return this.translate( 'Start', { context: 'verb' } );
 		}
 
 		if ( includes( cancelStates, importerState ) ) {
-			return this.translate( 'Cancel', { context: 'verb' } );
+			return this.translate( 'Close', { context: 'verb' } );
```

Comment:
Alternate string suggestion: 
* `translate( 'Close', { context: 'verb, to Close a dialog'} )`&mdash; translations: **23**. *ES Score: 6.60*

[PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/remove-stop-during-import)